src/utils/ethers-utils.ts

```diff
@@ -1,5 +1,19 @@
 import type { TransactionReceipt } from '@ethersproject/abstract-provider/lib'
+import { ErrorCode } from '@ethersproject/logger'
+
+// https://docs.ethers.io/v5/api/providers/types/#providers-TransactionResponse
+export enum EthersTxReplacedReason {
+  repriced = 'repriced',
+  cancelled = 'cancelled',
+  replaced = 'replaced',
```

Comment:
@iamacook do you know if the ethers package exports these?

Reply:
They're [just strings](https://github.com/ethers-io/ethers.js/blob/master/packages/providers/src.ts/base-provider.ts#L1372-L1377) sadly.